src/main/java/Todo.java

```diff
@@ -0,0 +1,34 @@
+public class Todo extends Task{
+
+    static Todo from(String s) throws InvalidDescriptionException {
+        //Expects a string in the format "todo <description>"
+
+        //get rid of the command
+        String description = utils.discardFirstWord(s.trim()).trim();
+
+        if (!description.isEmpty()){
+            return new Todo(description);
+        }else {
+            throw new InvalidDescriptionException("The description of a todo cannot be empty.");
+        }
+    }
+
+    public Todo(String description) {
+        super(description);
+    }
+
+    @Override
+    public void markAsDone() {
+        super.markAsDone();
+    }
+
+    @Override
+    public void markAsUndone() {
+        super.markAsUndone();
+    }
```

Comment:
Consider to remove these lines as they have same behaviour with the superclass of them.